array-methods/map.ts

```diff
@@ -0,0 +1,7 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+
+const sale = prices.map((num) => ({
+  price: num,
+  salePrice: num / 2,
+}));
+console.log(sale);
```

Comment:
There should be two different variables you create using `map`. So you should have another few lines here that convert all the numbers utilizing `toFixed`